app/preparing_knights/create.py

```diff
@@ -0,0 +1,52 @@
+from app.equipment.armour import Armour
+from app.equipment.potion import Potion
+from app.equipment.weapon import Weapon
+from app.preparing_knights.knight import Knight
+
+
+def create_armour(knight: dict) -> list[Armour]:
+    list_armour = knight.get("armour")
+    return [
+        Armour(armour["part"], armour["protection"])
+        for armour in list_armour
+    ]
+
+
+def create_weapon(knight: dict) -> Weapon:
+    weapon = knight.get("weapon")
+    return Weapon(weapon["name"], weapon["power"])
+
+
+def create_potion(knight: dict) -> Potion | None:
+    if knight.get("potion"):
+        potion = knight.get("potion")
+        return Potion(potion["name"], potion["effect"])
+
+
+def create_knights(knight: dict) -> Knight:
```

Comment:
Since only one knight is created, it would be better to name this function 
```suggestion
def create_knight(knight: dict) -> Knight:
```